apps/transport/lib/transport_web/live/backoffice/edit_dataset_live.ex

```diff
@@ -0,0 +1,249 @@
+defmodule TransportWeb.EditDatasetLive do
+  use Phoenix.LiveView
+  use Phoenix.HTML
+  import TransportWeb.Gettext, only: [dgettext: 2]
+  alias DB.Dataset
+  import TransportWeb.Router.Helpers
+  alias TransportWeb.InputHelpers
+
+  def render(assigns) do
+    ~H"""
+    <.form
+      :let={f}
+      for={:form}
+      action={@form_url}
+      phx-change="change_dataset"
+      phx-submit="save"
+      phx-trigger-action={@trigger_submit}
+    >
+      <h1>
+        <%= if is_nil(@dataset) do %>
+          <%= dgettext("backoffice", "Add a dataset") %>
+        <% else %>
+          <%= dgettext("backoffice", "Edit a dataset") %>
+        <% end %>
+      </h1>
+      <%= unless is_nil(assigns[:dataset]) do %>
+        <div class="pb-24">
+          <i class="fa fa-external-link-alt"></i>
+          <%= link(dgettext("backoffice", "See dataset on website"), to: dataset_path(@socket, :details, @dataset.id)) %>
+        </div>
+      <% end %>
+
+      <div class="pt-24">
+        <%= InputHelpers.text_input(f, :url,
+          placeholder: dgettext("backoffice", "Dataset's url"),
+          value:
+            if not is_nil(@dataset) do
+              Dataset.datagouv_url(@dataset)
+            else
+              ""
+            end
+        ) %>
+
+        <%= if assigns[:datagouv_infos] do %>
+          <% dataset_datagouv_id = @datagouv_infos[:dataset_datagouv_id] %>
+
+          <div class="pt-12 pb-12">
+            <%= if is_nil(dataset_datagouv_id) do %>
+              Impossible de trouver ce jeu de données sur data.gouv
+            <% else %>
+              <div>Jeu de données <strong>"<%= @datagouv_infos[:datagouv_title] %>"</strong></div>
+              <div class="pt-12">
+                Son identifiant data.gouv est <strong><%= @datagouv_infos[:dataset_datagouv_id] %></strong>
```

Comment:
```suggestion
                Son identifiant data.gouv est <code class="inline"><%= @datagouv_infos[:dataset_datagouv_id] %></code>
```

suggestion et sans tester

Reply:
fait dans https://github.com/etalab/transport-site/pull/2940/commits/fbe81a2b51f7e29d2cf2b24d45dde4056f68fd88 (les éléments code en html sont inline de base, j'ai pas rajouté de classe.)